content/quickstart-setup-basic-environment.rst

```diff
@@ -13,22 +13,23 @@ ______________
 Installation
 _____________________
 
-1. Create a conda environment to install all dependencies.
+1. Create and activate conda environment:
 
 .. code-block:: bash
 
     conda create --name IMPROVE python=3.9.16
     conda activate IMPROVE
 
-2. Install **Supervisor** from GitHub and add executables to your search path. 
+2. Setup **Supervisor** from GitHub and add executables to your search path. 
 
 .. code-block:: bash
 
     git clone https://github.com/ECP-CANDLE/Supervisor.git
     git checkout develop
```

Comment:
Might be better to move "git checkout develop" to after you cd in Supervisor repo folder. Otherwise you get this message they may be confusing to users:

`supervisor_env) ac.sejones@lambda0:~$ git clone https://github.com/ECP-CANDLE/Supervisor.git
Cloning into 'Supervisor'...
remote: Enumerating objects: 21937, done.
remote: Counting objects: 100% (5340/5340), done.
remote: Compressing objects: 100% (1672/1672), done.
remote: Total 21937 (delta 3673), reused 5251 (delta 3616), pack-reused 16597
Receiving objects: 100% (21937/21937), 46.67 MiB | 41.77 MiB/s, done.
Resolving deltas: 100% (15353/15353), done.
Updating files: 100% (970/970), done.
(supervisor_env) ac.sejones@lambda0:~$ git checkout develop
fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
(supervisor_env) ac.sejones@lambda0:~`

Reply:
Changed, thanks